Example/Tests/JSONApiTesting.swift

```diff
@@ -61,12 +47,20 @@ class JSONApiTesting: XCTestCase {
 
     final class Author: PSJSONApiModel {
 
+        open var name: String = ""
+        open var age: Int = 0
+        open var gender: String = ""
+        open var articles: [Articles] = []
         override class var modelName: String {
             return "authors"
         }
 
         override func register(attributes: inout [PSJSONAPIProperty], andRelationships relationships: inout [PSJSONAPIProperty]) {
-
+            attributes.append(PSAttribute(property: &self.name, jsonKey: "name"))
```

Comment:
Yes, its passing a pointer into the attribute objects. It is so that we can then use PSAttribute and PSToOne/ToMany to parse the JSON and change the value of the top level swift values in the model. I did this instead of using KVO to change the values because only objects that are objc compatible are KVO compliant